backend/Models/auth/verifyEmail.js

```diff
@@ -0,0 +1,39 @@
+const {pool} = require('../db');
+const jwt = require('jsonwebtoken');
+const fs = require('fs');
+
+/**
+ *
+ * @param {String} accessToken
+ * @return {Promise}
+ */
+function verifyEmail(accessToken) {
+  return new Promise((resolve, reject) => {
+    const path = require('path');
+    const pubKey = fs.readFileSync(path.resolve('rsa_secret.pub'));
+    jwt.verify(accessToken, pubKey, (error, decoded) => {
+      if (error) {
+        return reject(error);
+      }
+      let query = `UPDATE users SET verified=? `;
+      let arr = [1];
+      if (decoded.emailId) {
```

Comment:
Why are we using this?

Reply:
What if emailid is not stored? someone created fake token? Or used one issued in login API?

Reply:
This is for updating the email ..so whenever there is email field in update route, the email will be signed along with username as a payload ...so he has to verify this email and then it has to be stored right..thats why..for fake token secret will be not verified right